### DISCUSSION TITLE
Remove spoofDPI

### DISCUSSION THREAD

**Discussion by senyaghtbs** [xvzc/SpoofDPI#102] [Q&A]
Hi, how i can remove spoof? 
Today i came to work and some sites that works with our vpn doesnt work now..

**Comment by nikitintima46** [comment]
I had the same problem. To fix this, I uninstalled the WiFi service on Mac OS and created it again.

**Comment by PoorMouse** [comment]
rm -rf ~/.spoof-dpi

**Comment by okgo2012** [comment]
Напишите, пожалуйста, алгоритм удаления и отмены всех действий после установки скрипта. Перестали работать некоторые функции в браузере и расширениях. А сам скрипт проблемы не решил на m1 mac

**Reply by PoorMouse** [reply]
Программа просто копируется в папку .spoof-dpi в вашем домашнем каталоге. Она никуда не "инсталлируется". Достаточно остановить ее (если запущена) и удалить этот каталог.

**Reply by nikitintima46** [reply]
Мне не помогло обычное удаление каталога. Чтобы все заработало снова, я на Mac OS удалил сервис WiFi и создал его вручную. Для этого необходимо перейти в настройки - Сеть. Нажать правой кнопкой мыши на WiFi и удалить его. После чего создать новый.

**Reply by eboboshka** [reply]
@nikitintima46 у вас, скорее всего, был неправильно остановлен процесс из-за чего в настройках подключения wifi остался прописан proxy localhost. В следующий раз, проще зайдите в настройки подключения и убрать галочки напротив proxy

**Comment by Leny130** [comment]
Не открываются мессенджеры, как исправить?
